opennms-doc/guide-development/src/asciidoc/text/rest/node_links.adoc

```diff
@@ -0,0 +1,27 @@
+[[Node-Links]]
+==== Node Links
+
+===== GETs (Reading Data)
+NOTE: This API is v2-only
+
+[options="header", cols="5,10"]
+|===
+| Resource                                                          | Description
+| `/enlinkd/{nodeId}`  | Retrieve all types of links for a given node
+| `/enlinkd/lldplinks/{nodeId}`  | Retrieve lldp link for a given node
+| `/enlinkd/bridgelinks/{nodeId}`| Retrieve bridge link for a given node
+| `/enlinkd/cdplinks/{nodeId}`   | Retrieve cdp link for a given node
+| `/enlinkd/ospflinks/{nodeId}`  | Retrieve ospf link for a given node
+| `/enlinkd/isislinks/{nodeId}`  | Retrieve isis link for a given node
+| `/enlinkd/lldpelems/{nodeId}`  | Retrieve lldp element for a given node
+| `/enlinkd/cdpelems/{nodeId}`   | Retrieve cdp element for a given node
```

Comment:
```suggestion
| `/enlinkd/cdpelems/{nodeId}`   | Retrieve CDP element for a given node.
```

Reply:
Is there such a thing as a "bridge" element? We have it in the list of links above, but it is not repeated here as an element, while all the other protocols are.

Reply:
I'm not sure about this one. This rest api follows what the existing linkdnode.jsp does to make sure they are synced. Not sure if the bridge element is skipped intentionally or just by accident. Need someone to confirm.

Reply:
It's not there in jsp file but it's available in `EnLinkdElementFactoryInterface` .  Let's add that.

Reply:
Okay, will add it.

Reply:
Updated.